trade-finance-manager-ui/templates/case/tasks/task.component-test.js

```diff
@@ -56,13 +56,13 @@ describe(page, () => {
   });
 
   it('should render deal subheading', () => {
-    const expected = `${params.deal.details.ukefDealId} ${params.deal.submissionDetails.supplierName} > ${params.deal.submissionDetails.buyerName}`;
+    const expected = `View deal${params.deal.details.ukefDealId} details ${params.deal.submissionDetails.supplierName} > ${params.deal.submissionDetails.buyerName}`;
```

Comment:
I think we need space after `View deal`
We have space in other similar places, for example `View facility `.

Reply:
Added space after `View deal` for `trade-finance-manager-ui\templates\case\tasks\task.component-test.js` `const expected = `View deal ${params.deal.details.ukefDealId} details ${params.deal.submissionDetails.supplierName} > ${params.deal.submissionDetails.buyerName}`;`